tests/psi4numpy/rhf_gradient/input.py

```diff
@@ -0,0 +1,163 @@
+"""
+This script calculates nuclear gradients of RHF Wavefunction using
+gradients of one and two electron integrals obtained from PSI4. 
+
+Reference: "Derivative studies in hartree-fock and møller-plesset theories",
+J. A. Pople, R. Krishnan, H. B. Schlegel and J. S. Binkley
+DOI: 10.1002/qua.560160825
+"""
```

Comment:
I'm not sure what about this file (and hess) is offending py27, but according to travis, looks like something is. maybe build a 27 locally to find out why. or maybe that's what [this](https://github.com/ashutoshvt/psi4/blob/b3c4612f9ef0657d8d6e9e4cbdbcf99136bebed4/tests/libxc/input.dat#L135) is for.

Reply:
Fixed it. Python 2 was not able to handle the non-ascii character ø in the reference.